src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -1197,7 +1197,12 @@ public boolean fileMetadataExtractableFromNetcdf(DataFile dataFile, Path tempLoc
             StorageIO<DataFile> storageIO;
             try {
                 storageIO = dataFile.getStorageIO();
-                storageIO.open();
+                try {
```

Comment:
If I'm reading the code correctly, you are running this code on all the uploaded files again. Instead of just the files of the applicable type. I am indeed getting a `NetcdfFiles.open()` exception on every file upload. We should avoid doing this.

Reply:
I am confused by what's being done here. Your comment above this method says "Inspired by fileMetadataExtractable, above" - so what's the reason not to be doing the same thing as in fileMetadataExtractable() which is simply checking the mime type?

Reply:
Makes sense, just like the "canIngestAsTabular()" method checks for a whole list of ingestable mime types.